dao-go/environment_test.go

```diff
@@ -223,19 +219,25 @@ func SetupEnvironmentWithFlags(t *testing.T, addFakePeriods, addFakeMembers bool
 	assert.NilError(t, err)
 
 	t.Log("Deploying SeedsExchange contract to 		: ", env.SeedsExchange)
-	_, err = eostest.SetContract(env.ctx, &env.api, env.SeedsExchange, exchangeWasm, exchangeAbi)
+	_, err = eostest.SetContract(env.ctx, &env.api, env.SeedsExchange, exchangePrefix+"wasm", exchangePrefix+"abi")
 	assert.NilError(t, err)
 	loadSeedsTablesFromProd(t, &env, "https://api.telos.kitchen")
 
 	t.Log("Deploying Events contract to 		: ", env.Events)
 	_, err = eostest.SetContract(env.ctx, &env.api, env.Events, monitorPrefix+"wasm", monitorPrefix+"abi")
 	assert.NilError(t, err)
 
-	_, err = dao.CreateRoot(env.ctx, &env.api, env.DAO)
+	eostest.Pause(time.Second * 10, "Build block...", "")
+	_, err = CreateRoot(env.ctx, &env.api, env.DAO)
 	assert.NilError(t, err)
+	eostest.Pause(time.Second * 10, "Build block...", "")
 	env.Root, err = docgraph.LoadDocument(env.ctx, &env.api, env.DAO, "52a7ff82bd6f53b31285e97d6806d886eefb650e79754784e9d923d3df347c91")
-	assert.NilError(t, err)
 
+	eostest.Pause(time.Second * 10, "Build block...", "")
```

Comment:
I also discovered a change somewhere else (making tests fail ATM). It's not waiting for voting to finish and fails.

Reply:
Yeah I had some assignment tests (assignment_test.go) with that problem since previously there wasn't a check for voting period when closing a proposal. I changed some of them to wait for the full voting duration but there might be others pending. I'll update those later.